velero-plugin-for-gcp/object_store.go

```diff
@@ -39,6 +39,7 @@ const (
 	kmsKeyNameConfigKey      = "kmsKeyName"
 	serviceAccountConfig     = "serviceAccount"
 	credentialsFileConfigKey = "credentialsFile"
+	endpointConfigKey        = "endpoint"
```

Comment:
should we maybe explicitly call it storeEndpoint?
That way if there are other endpoints which are required in future, we don't cause confusion.

Reply:
Good point. It's possible the snapshot endpoint also needs modification.